client/src/App.js

```diff
@@ -1,20 +1,35 @@
 import React from 'react';
 import './App.css';
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
-
+import { Grid } from '@material-ui/core';
 import NewMember from './components/LayoutComponents/Dashboard/NewMember';
 
 import Home from './components/LayoutComponents/Home';
-// import Dashboard from './components/LayoutComponents/Dashboard';
+import Dashboard from './components/LayoutComponents/Dashboard';
 
 function App() {
 	return (
 		<Router>
-			<Switch>
-				<Route exact path="/dashboard" component={NewMember} />
-				<Route path="/" exact component={Home} />
-				{/* <Route path="/dashboard" exact component={Dashboard} /> */}
-			</Switch>
+			<Grid direction="column" container>
```

Comment:
Why Grid in the router!!!!

Reply:
its a standard template for all of us to make sure we are having the same space on the page